lib/routes/account.js

```diff
@@ -959,10 +1003,13 @@ module.exports = function (
         }
 
         function createResponse() {
-          return {
-             email: sessionToken.email,
-             verified: sessionToken.emailVerified
-           }
+          return db.sessionTokenVerified(sessionToken.id)
+          .then( function(token) {
+            return {
+              email: token.email,
+              verified: !!(token.emailVerified && ! token.tokenVerificationId)
```

Comment:
I'd like to see us send additional fields here to indicate *what* is not verified, to help the client code give a better experience.  Does it make sense to send similar `challenge*` fields to what we send elsewhere?  Or perhaps additional `sessionVerified` and `emailVerified` fields?